api/src/models.py

```diff
@@ -9,13 +11,26 @@
 class Company(SQLModel, table=True):
     """Model representing a Company in the database."""
 
-    id: int | None = Field(primary_key=True, nullable=False, default=None)
-    created: datetime | None = Field(nullable=False, default_factory=datetime.now)
-    is_bankrupt: bool | None = Field(nullable=False, default=False)
-    networth: float | None = Field(nullable=False, default=0)
+    id: int | None = Field(primary_key=True, default=None)
```

Comment:
Default value cannot be `null` if it is `primary_key` because primary key requires unique value and cannot be `null`

Reply:
Has to do with how SQLModel works. Here is the link to the site: [here](https://sqlmodel.tiangolo.com/tutorial/create-db-and-table/#primary-key-id)

In summary, when creating the model, the ID is unknown. Only after it has been commited to the database and refreshed will the ID be present. That is the reasoning behind why it is `None` despite there being a `primary_key`

Reply:
💀ok, the design is weird enough thx for the info